tree/dataframe/test/dataframe_vecops.cxx

```diff
@@ -95,8 +95,7 @@ void MakeTreeWithBools(const std::string &treename, const std::string &fname)
    }
    t.Fill();
 
-   t.Write();
-   f.Close();
+   f.WriteObject(&t, t.GetName());
```

Comment:
Yep, both are good. I still prefer the more explicit `WriteObject`

Reply:
> f.WriteObject(&t, t.GetName());

(This is not essential since this is a test but) This sends the wrong message that `f` and `t.GetDirectory()->GetFile()` (aka the owner of the `TTree`) can routinely be different.  Try to read the files if one does:
```
TFile f1("file1.root", "RECREATE");
TFile f2("file2.root", "RECREATE");
TTree t("t", "t");
... fill some data;
f1.WriteObject(&t, t.GetName());
```
This can be made to work but `file2.root` has to be explicit recorded as the holder of data.  For reading such a file, at (read or write time) `TBranch::SetFile` needs to be called for each branches.

Reply:
```suggestion
   f.Write();
```